lib/pages/home/home_controller.dart

```diff
@@ -1,15 +1,33 @@
 import 'package:get/get.dart';
+import 'package:intl/intl.dart';
 import 'package:survey/managers/user_manager.dart';
 
+const HOME_DISPLAY_DATE_FORMAT = 'EEEE, MMMM dd';
+const AVATAR_PLACE_HOLDER =
```

Comment:
Can we save it into our resources? 🤔

Reply:
we can, a similar idea so I just demonstrate it quickly as the placeholder, for now, 😅  I can update if you guys wanted

Reply:
<img src="https://user-images.githubusercontent.com/13435717/121670412-987c6280-cad7-11eb-9334-3b64c7621c9b.gif" width=350 />

updated